software_setup/setup_nuke.cmd

```diff
@@ -17,19 +17,24 @@ rem SET NUKE_AF_RENDERTHREADS=2
 rem Nuke render launcher:
 SET NUKE_AF_RENDER=nuke
 
-For /F "Tokens=*" %%I in ('dir /b "C:\Program Files\Nuke*"') Do set "NUKE_DIR=C:\Program Files\%%I"
-rem echo %NUKE_DIR%
-For /F "Tokens=*" %%I in ('dir /b "%NUKE_DIR%\Nuke*.*.exe"') Do set "NUKE_EXEC=%NUKE_DIR%\%%I"
+
+IF NOT DEFINED NUKE_LOCATION (
+   For /F "Tokens=*" %%I in ('dir /b "C:\Program Files\Nuke*"') Do set "NUKE_LOCATION=C:\Program Files\%%I"
```

Comment:
It is good to have a consistency in the env var nameing, but there might be users depending on the ``NUKE_DIR`` env var. I would suggest also setting this env var.